### PR TITLE
[Update] Japanese translation of docs/cli.md and docs/cli_commands.md

### PR DESCRIPTION
**Pull request by mtei**
## Description

Update Japanese translation.
 * docs/ja/cli.md
 * docs/ja/cli_commands.md

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@umi-umi @shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

**Comment by noroadsleft** [comment]
Looks good to me so far. Are you planning on doing `docs/cli_configuration.md` and `docs/cli_development.md` in a later PR?

**Comment by mtei** [comment]
>  Are you planning on doing `docs/cli_configuration.md` and `docs/cli_development.md` in a later PR?

* docs/cli_configuration.md
  This has already been translated into Japanese. https://github.com/qmk/qmk_firmware/blob/master/docs/ja/cli_configuration.md

* docs/cli_development.md
  It is undergoing translation review in #9513.

Thank you.

**Comment by noroadsleft** [comment]
Whoops, I checked your source branch here but I didn't look at QMK's master :rofl:

File: docs/ja/cli_commands.md [hunk 1]
```diff
@@ -1,48 +1,18 @@
...
-```
-qmk cformat -b branch_name
-```
+# ユーザー用コマンド
 
 ## `qmk compile`
 
 このコマンドにより、任意のディレクトリからファームウェアをコンパイルすることができます。<https://config.qmk.fm> からエクスポートした JSON をコンパイルするか、リポジトリ内でキーマップをコンパイルするか、現在の作業ディレクトリでキーボードをコンパイルすることができます。
 
+このコマンドはディレクトリ対応です。キーボードやキーマップのディレクトリにある場合、自動的に KEYBOARD や KEYMAP を入力します。
```
[the remark this answers is not in the document]
Comment (mtei) [reply docs/ja/cli_commands.md:14]: 最初「認識します」と訳して、もっと意訳してしまえと「対応です」に変えたんですが、、、

どうしても「認識します」にしたいですか？

[the remark this answers is not in the document]
Comment (shelaf) [reply docs/ja/cli_commands.md:14]: QMKのディレクトリの構造に対応しています、で書くなら対応でも分かるのですが、そもそも特定のディレクトリで実行するコマンドなので、ディレクトリ対応とは？みたいにならないかなという懸念があります。

**Comment by mtei** [comment]
I think this is ready to merge.

**Pull request merged**